### ISSUE TITLE
Inconsistent TAB behaviour

### ISSUE THREAD

**Issue by paulossilva** [remy/txt2bas#14]
In the online editor, pressing TAB for a single line gives a four-space indentation, but when used with a multiple line selection it gives a two-space indentation.

I'm using code without line numbers as I use the #autoline directive

**Comment by remy** [comment]
Oh, I'm not aware of the #autoline directive (I'm pretty sure my code is ignorant of the feature).

To clarify what you're reporting, when you've got multiple lines selected and *then* you hit tab, it's indenting with 2 spaces.

**Comment by paulossilva** [comment]
That's right, when I select multiple lines in the online editor, I get two spaces for indenting by hitting tab instead of four spaces (which is what I get for a single line indenting). I prefer 2 spaces instead of 4 by the way :-)

**Comment by remy** [comment]
Not to self, can also replicate with this line:

```
20 PRINT "hello world"
```

Place cursor after `20` and tab=2 spaces, after `PRINT` tab=tab char. Weird.
